<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>TeacherDialog</class>
	<widget class="QDialog" name="TeacherDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>500</width>
				<height>400</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Преподаватель</string>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<!-- Таблица предметов -->
			<item>
				<widget class="QTableWidget" name="tblSubjects">
					<property name="columnCount">
						<number>2</number>
					</property>
					<property name="rowCount">
						<number>0</number>
					</property>
					<column>
						<property name="text">
							<string>Группа</string>
						</property>
					</column>
					<column>
						<property name="text">
							<string>Предмет</string>
						</property>
					</column>
				</widget>
			</item>
			<!-- Кнопки управления -->
			<item>
				<layout class="QHBoxLayout" name="horizontalLayout">
					<item>
						<widget class="QPushButton" name="btnAdd">
							<property name="text">
								<string>Добавить</string>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QPushButton" name="btnRemove">
							<property name="text">
								<string>Удалить</string>
							</property>
						</widget>
					</item>
					<item>
						<spacer name="horizontalSpacer">
							<property name="orientation">
								<enum>Qt::Horizontal</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>40</width>
									<height>20</height>
								</size>
							</property>
						</spacer>
					</item>
				</layout>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
